### PR TITLE
New 9 create competitions page kovalchuk

### PR DESCRIPTION
**Pull request by Maksson**

### CODE REVIEW

**Review by MetalMari — APPROVED** [review]

**Review by s0fko — APPROVED** [review]

File: front-end/src/Components/pages/competitionPage/Block.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Navbar, Container, Row, } from 'react-bootstrap';
+
+import MainBlockCard from './MainBlockCard';
+
+export default function Block() {
+    return (
+        <>
```
Comment (vvelizariy) [inline front-end/src/Components/pages/competitionPage/Block.js]: Is there any sense wrapping Navbar into <> ?

File: front-end/src/Components/pages/competitionPage/Functional.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+export default function Functional() {
+    return (
+        <>
+            <Navbar collapseOnSelect expand="md" className="grey type" >
+                <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+                <Navbar.Collapse id="responsive-navbar-nav">
+                    <LeftFunctional />
+                    <RightFunctional />
+
```
Comment (vvelizariy) [inline front-end/src/Components/pages/competitionPage/Functional.js]: Remove empty line

File: front-end/src/Components/pages/competitionPage/Functional.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Navbar } from 'react-bootstrap';
+
+import LeftFunctional from './LeftFunctional';
+import RightFunctional from './RightFunctional';
+
+
+export default function Functional() {
+    return (
+        <>
```
Comment (vvelizariy) [inline front-end/src/Components/pages/competitionPage/Functional.js]: Same, if you return single component, no sense wrapping into <>

File: front-end/src/Components/pages/competitionPage/HeaderBlock.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    constructor(props) {
+        super(props);
+    }
+    render() {
+        const { card } = this.props
+        return (
+            <>
+            <div className="blue">
+                <Card.Header >
+
```
Comment (vvelizariy) [inline front-end/src/Components/pages/competitionPage/HeaderBlock.js]: useless empty lines

File: front-end/src/Components/pages/competitionPage/RightBlock.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    render() {
+        const { card } = this.props
+        return (
+            <>
+                <Card bg="light" style={{ width: '15rem' }} >
+                    <Card.Body style={{ height: '17rem' }}>
+                        <Card.Img variant="top" style={{ height: '10rem' }} src={card.src} />
+                        <Card.Text className="text-center m-3" ><h5> {card.company_name} </h5>  </Card.Text>
```
Comment (vvelizariy) [inline front-end/src/Components/pages/competitionPage/RightBlock.js]: Format as tree
```
 <Card.Text className="text-center m-3" >
    <h5> {card.company_name} </h5>
```

File: front-end/src/Components/pages/competitionPage/RightFunctional.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Nav, Button, Form, FormControl, Dropdown } from 'react-bootstrap';
+
+
```
Comment (vvelizariy) [inline front-end/src/Components/pages/competitionPage/RightFunctional.js]: Why four empty lines? 2 is enough)

File: front-end/src/Components/pages/competitionPage/Block.js (hunk 1 above)
Comment (Maksson) [reply front-end/src/Components/pages/competitionPage/Block.js]: No, I remove it

File: front-end/src/Components/pages/competitionPage/Functional.js (hunk 3 above)
Comment (Maksson) [reply front-end/src/Components/pages/competitionPage/Functional.js]: ok

File: front-end/src/Components/pages/competitionPage/Functional.js (hunk 2 above)
Comment (Maksson) [reply front-end/src/Components/pages/competitionPage/Functional.js]: ok, done

File: front-end/src/Components/pages/competitionPage/HeaderBlock.js (hunk 4 above)
Comment (Maksson) [reply front-end/src/Components/pages/competitionPage/HeaderBlock.js]: deleted

File: front-end/src/Components/pages/competitionPage/RightFunctional.js (hunk 6 above)
Comment (Maksson) [reply front-end/src/Components/pages/competitionPage/RightFunctional.js]: resolved

File: front-end/src/Components/pages/competitionPage/RightBlock.js (hunk 5 above)
Comment (Maksson) [reply front-end/src/Components/pages/competitionPage/RightBlock.js]: Was changed

**Review by Viktorika-p — APPROVED** [review]

**Pull request merged**